CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 1.7.0 / 2017-06-06
+
+* [CHANGE] Compress remote storage requests and responses.
+* [FEATURE] Add OpenStack service discovery.
+* [FEATURE] Add ability to limit Kubernetes service discovery to certain namespaces.
+* [FEATURE] Add metric for discovered number of Alertmanagers.
+* [ENHANCEMENT] Print system information (uname) on start up.
+* [ENHANCEMENT] Properly ellide secrets in config.
```

Comment:
I don't mind too much, but I'd expect CHANGE only for actually breaking changes. As we don't have an API for the config yet, I wouldn't list pure UI changes.